src/mono/wasm/wasm.proj

```diff
@@ -230,4 +230,28 @@
           SkipUnchangedFiles="true" />
   </Target>
 
+  <Target Name="RunTests">
+    <ItemGroup Condition="'$(VSEngine)' == ''">
+      <VSEngines Include="V8" />
+      <VSEngines Include="SpiderMonkey" />
+      <!-- <VSEngines Include="JavaScriptCore" /> -->
+    </ItemGroup>
+    <ItemGroup Condition="'$(VSEngine)' != ''">
+      <VSEngines Include="$(VSEngine)" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(Library)' == ''">
+      <Libraries Include="$(RepoRoot)src\libraries\*\tests\*.csproj">
+        <Engine>%(VSEngines.Identity)</Engine>
+      </Libraries>
+    </ItemGroup>
+    <ItemGroup Condition="'$(Library)' != ''">
+      <Libraries Include="$(RepoRoot)src\libraries\$(Library)\tests\">
+        <Engine>%(VSEngines.Identity)</Engine>
+      </Libraries>
+    </ItemGroup>
+
+    <Exec Command="dotnet build %(Libraries.RootDir)%(Libraries.Directory) /p:JSEngine=%(Libraries.Engine) /t:Test /p:TargetOS=Browser /p:TargetArchitecture=wasm" />
```

Comment:
This seems like it is only for non-DOM scenarios. Are we also going to be enabling DOM scenarios? The Crypto APIs will not be able to run on non-DOM environments. I assume this is complex given the need for some browser driver being installed.

See https://github.com/dotnet/runtime/pull/49511

Reply:
Yes, this is now only for stand alone engines. I plan to add target to run the tests in browser on windows in future PR.